core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -270,6 +270,12 @@ package object config {
     .longConf
     .createWithDefault(4 * 1024 * 1024)
 
+  private[spark] val IGNORE_EMPTY_SPLITS = ConfigBuilder("spark.files.ignoreEmptySplits")
```

Comment:
This config should be made internal, and the name should be improved because it's not about spark files.

Reply:
I'll send a follow-up PR to fix this.